Examples/SpeechRecognition/SpeechRecognition/SpeechRecognition.swift

```diff
@@ -98,15 +98,14 @@ let appReducer = Reducer<AppState, AppAction, AppEnvironment> { state, action, e
       request.shouldReportPartialResults = true
       request.requiresOnDeviceRecognition = false
       return environment.speechClient.recognitionTask(SpeechRecognitionId(), request)
-        .catchToEffect()
-        .map(AppAction.speech)
+        .catchToEffect(AppAction.speech)
 
     @unknown default:
       return .none
     }
   }
 }
-.debug()
+  .debug(actionFormat: .labelsOnly)
```

Comment:
the actions for this demo app hold a lot of data so it really drowns out the state changes. so just switching to this for now.